docs/docs/server-administration/dns.md

```diff
@@ -63,18 +62,20 @@ This way you can set up Master -> Slave or Master <-> Master <-> Master cluster.
 
 There is no limitation on how to chain DNS servers.
 
-### Master -> Slave DNS cluster with the Hestia API 
+### Master -> Slave DNS cluster with the Hestia API
 
 Preparing your **Slave** server(s):
 
 1. Whitelist your master server IP in **Configure Server** -> **Security** -> **Allowed IP addresses for API**
 2. Enable API access for admins (or all users).
 3. Create an API key under the **admin** user with at least the **sync-dns-cluster** permission.
 4. Create a new DNS sync user as follows:
--  Username of "dns-user"
--  Has email address (something generic)
--  Has the role `dns-cluster`
+
+- Username of "dns-user"
```

Comment:
Username doesn't matter can be every thing. 

As long the role is set to dns-cluster